usagov/usagov-2021
# USAGOV-1747-wizard-es-topnav: Create twig logic to show an ES top nav on wizard taxonomy pages.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1747

## Description

Create wizard-top-nav-es twig template and add logic to existing twig templates to show this on ES wizard pages.

## Type of Changes

- [ X ] New Feature
- [ X ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like

## Testing Instructions


1. Visit a handful of wizard taxonomy pages in both english and spanish.
2. The language of the top navigation items (e.g "All topics and services") matches that of the language of the term.
3. Go to a wizard taxonomy page with an existing language toggle value, or go into the edit form of a wizard taxonomy page and set a language toggle value and save.
4. Utilize the Language switch link in the upper right.
5. The language of the page switches as expected.
6. Visit non-wizard taxonomy pages in both languages.
7. The non-wizard pages show the top navigation as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
8. Find the commit of this pull request.
9. Build and deploy the changes.
10. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
